gitmine/gitmine.py

```diff
@@ -1,17 +1,46 @@
+import click
+
 from typing import Optional
 
-import click
+try: 
+    from http.client import HTTPConnection
+except ImportError:
+    from httplib import HTTPConnection
 
 from gitmine.commands.config import config_command, get_or_create_github_config
 from gitmine.commands.get import get_command
 from gitmine.commands.go import go_command
+from gitmine.constants import LOGGER, LOGGER_PATH
 
+import logging
+import logging.config
+logging.config.fileConfig(LOGGER_PATH)
 
 @click.group()
+@click.option(
+    "--verbosity", "-v",
```

Comment:
verbose instead of verbosity is standard

Reply:
Idk where it's being set, but I don't think we should show the HTTPRequest information at the INFO level, only DEBUG. It's a lot of info.

Reply:
HTTPRequest information is just coming straight from the requests logger which we turn on manually at the moment. So that information is at the INFO level of the requests logger.
I'll turn that off for now until we can figure out how to pipe it down to DEBUG level